image_generation/requirements.txt

```diff
@@ -0,0 +1,6 @@
+--extra-index-url https://download.pytorch.org/whl/cpu
+torch==2.2.2+cpu; sys_platform != 'darwin'
+torch==2.2.2; sys_platform == 'darwin'  # Torch wheel for 2.2.2+cpu does not exist for macOS, using 2.2.2 instead
```

Comment:
I just realized. I'd expect torch to come with optimum. Can torch be removed?

Reply:
Duplicated dependencies removed keeping only diffusers in requirements